docs/src/managing-packages.md

```diff
@@ -133,6 +133,38 @@ Updating `~/.julia/environments/v1/Manifest.toml`
   [92a963f6] + ImportMacros v1.0.0 `git@github.com:fredrikekre/ImportMacros.jl.git#master`
 ```
 
+### Adding a subdir package
+
+If the package you want to add is not in the root of the repository, then you need to manually pass the `subdir` keyword to `Pkg.add`
```

Comment:
Maybe specify to "you want to add by URL". `SnoopCompileCore` is registered and can be installed by package name without saying anything about a subdir. Possibly also move this to after "Adding a local package" since `subdir` also applies to add by path.